src/label_studio_sdk/converter/imports/coco.py

```diff
@@ -270,7 +271,7 @@ def add_parser(subparsers):
         "--input",
         dest="input",
         required=True,
-        help="directory with COCO where images, labels, notes.json are located",
+        help="input COCO json file",
```

Comment:
fixed to be consistent with: (not discussed in the original issue though)
https://github.com/HumanSignal/label-studio-sdk/blob/a67c59c724aaed8e0abdd368e4dea07306c78398/src/label_studio_sdk/converter/imports/coco.py#L120